### ISSUE TITLE
Video - can it be trimmed?

### ISSUE THREAD

**Issue by ghost** [joshgordon/sermon-archive#3]
The video from Jan 19, 2014 - Title: A New Year, A Good Life

It was never trimmed - everything is included in the video - so it is private with a copyright claim. Is there a way to trim it to the sermon only?

https://studio.youtube.com/video/h-BI7gCyyl4/edit

**Comment by joshgordon** [comment]
I *thought* we re-uploaded a trimmed version...?

**Comment by ghost** [comment]
Does it have a new name?

-D
*Debbie Gordon*|Bookkeeper/Pastoral Assistant
Severna Park EP Church
Office 410.544.5013

**Comment by joshgordon** [comment]
Nope, looks like we didn't. It was probably a transfer over from ustream... Go ahead and do it, if you can't figure it out, I'll take a look.

**Comment by joshgordon** [comment]
Nope, looks like we didn't. It was probably a transfer over from ustream... Go ahead and do it, if you can't figure it out, I'll take a look.
